### DISCUSSION TITLE
Working with Multiple Checkboxes in PDF Form

### DISCUSSION THREAD

**Discussion by DamithaPerera** [Hopding/pdf-lib#1471] [Q&A]
I have two checkboxes in my pdf. They are "Ja" & "Nei".  But when i use `const fields = form.getFields()` this property to get checkbox names im getting only `name===> PDFCheckBox: dodJaNei`

So how can i use two checkboxes 

` const checkBoxxx = form.getCheckBox('dodJaNei')`
 ` checkBoxxx.check()`
  
  in above logic it checks the first checkbox but not the second check box.

**Comment by zmunk** [comment]
I have the exact same problem.

**Reply by DamithaPerera** [reply]
[https://github.com/Hopding/pdf-lib/issues/1185](https://github.com/Hopding/pdf-lib/issues/1185)
try this. I was abele to find the solutions from this.

**Reply by satyajitnayk** [reply]
@zmunk @DamithaPerera  This might help [check the checkboxes based on the export values when have same names](https://github.com/cantoo-scribe/pdf-lib/issues/32)
